src/App.js

```diff
@@ -12,6 +12,7 @@ import './App.css'
 import FaceRecognition from './components/FaceRecognition'
 import GitHub from './components/GitHub/GitHub'
 import { handleLogout } from './utils/session'
+import Loading from './components/Loading'
```

Comment:
Please cluster all of your imports by root directory at the top of each file.

```
react & other packages

./utils

./components

./Component.module.css
```

Separate each cluster with a new line.